webclient/core/repository_connector.py

```diff
@@ -791,34 +783,24 @@ def record_ident_change(self, ident_cely_old):
             raise IdentChangeFedoraError()
         base_url = f"{settings.FEDORA_PROTOCOL}://{settings.FEDORA_SERVER_HOSTNAME}:{settings.FEDORA_PORT_NUMBER}/rest/"
         ident_cely_new = self.record.ident_cely
-        data = f"INSERT DATA {{<> <http://purl.org/dc/terms/isReplacedBy> " \
-               f"'{base_url}{settings.FEDORA_SERVER_NAME}/record/{ident_cely_new}'}}"
-        headers = {
-            "Content-Type": "application/sparql-update"
-        }
-        url = self._get_request_url(FedoraRequestType.CHANGE_IDENT_CONNECT_RECORDS_1, ident_cely=ident_cely_old)
-        self._send_request(url, FedoraRequestType.CHANGE_IDENT_CONNECT_RECORDS_1, headers=headers, data=data)
         data = f"INSERT DATA {{<> <http://purl.org/dc/terms/replaces> " \
                f"'{base_url}{settings.FEDORA_SERVER_NAME}/record/{ident_cely_old}'}}"
         headers = {
             "Content-Type": "application/sparql-update"
         }
         url = self._get_request_url(FedoraRequestType.CHANGE_IDENT_CONNECT_RECORDS_2, ident_cely=ident_cely_new)
         self._send_request(url, FedoraRequestType.CHANGE_IDENT_CONNECT_RECORDS_2, headers=headers, data=data)
-        data = "INSERT DATA {<> <http://purl.org/dc/terms/type> 'renamed'}"
-        headers = {
-            "Content-Type": "application/sparql-update"
-        }
-        url = self._get_request_url(FedoraRequestType.CHANGE_IDENT_CONNECT_RECORDS_3, ident_cely=ident_cely_old)
-        self._send_request(url, FedoraRequestType.CHANGE_IDENT_CONNECT_RECORDS_3, headers=headers, data=data)
         headers = {
             "Slug": ident_cely_old,
             "Content-Type": "text/turtle"
         }
         data = f"@prefix ore: <http://www.openarchives.org/ore/terms/> . " \
                f"<> ore:proxyFor <info:fedora/{settings.FEDORA_SERVER_NAME}/record/{ident_cely_old}> ."
-        url = self._get_request_url(FedoraRequestType.CHANGE_IDENT_CONNECT_RECORDS_4, ident_cely=ident_cely_new)
+        url = self._get_request_url(FedoraRequestType.CHANGE_IDENT_CONNECT_RECORDS_4, ident_cely=ident_cely_old)
```

Comment:
@pesikj Tady prosím o kontrolu. Z kódu mi přijde, že se to stejně v daném volání nepoužije, ale to `ident_cely_new` tam bylo vyloženě matoucí.